File: spring_java_orm_sample/src/main/java/com/example/demo/jooq/entity/PurchaseInvoiceDetailRow.java
package com.example.demo.jooq.entity;

import lombok.Value;

@Value
public class PurchaseInvoiceDetailRow {
    Long id;
    Long purchaseInvoiceId;
    Long purchaseInvoiceDetailBookId;
    Integer purchaseInvoiceDetailUnitPrice;
    Integer purchaseInvoiceDetailQuantity;
    Long purchaseInvoiceDetailAmount;
    Long version;
}

File: codex_poting/kotlin/src/main/kotlin/com/example/demo/jooq/entity/PurchaseInvoiceDetailRow.kt
package com.example.demo.jooq.entity

data class PurchaseInvoiceDetailRow(
    val id: Long?,
    val purchaseInvoiceId: Long?,
    val purchaseInvoiceDetailBookId: Long?,
    val purchaseInvoiceDetailUnitPrice: Int?,
    val purchaseInvoiceDetailQuantity: Int?,
    val purchaseInvoiceDetailAmount: Long?,
    val version: Long?
)
